pkgs/top-level/all-packages.nix

```diff
@@ -19814,6 +19814,8 @@ in
 
   virtinst = callPackage ../applications/virtualization/virtinst {};
 
+  virtscreen = callPackage ../tools/admin/virtscreen {};
```

Comment:
I think this application has to be `python3` so you might as well do `python3Packages.callPackage`

This does https://github.com/NixOS/nixpkgs/blob/9d8ccfe00f1c3b6122fa53529abbe2be381fccea/pkgs/top-level/python-packages.nix#L32

Basically everything that is in scope for derivations at  pkgs/development/python-modules will be for virtscreen.